doc/configuration.rst

```diff
@@ -797,6 +797,26 @@ Such device could be a signal generator.
 Used by:
   - `PyVISADriver`_
 
+IPVideo
+~~~~~~~~~~~~~
+
+A :any:`IPVideo` resource describes a IP video stream. `http` and `https`
+transport protocols are supported.
+
+.. code-block:: yaml
+
+   IPVideo:
+     uri: 'http://192.168.110.11/0.ts'
+
+Used by:
+  - `IPVideoDriver`_
+
+NetworkIPVideo
+~~~~~~~~~~~~~~~
+
+A :any:`NetworkIPVideo` resource describes a :any:`IPVideo` resource available
+on a remote computer.
+
```

Comment:
It seem GitHub lost my comment...
Only IPVideo (maybe named HTTPVideoStream?) would be needed, similar to ModbusTCPCoil or OneWirePIO, where the exporter is not involved and just passes through the information.

Reply:
Ah, I though the Network version would be needed for SSH proxying in isolated mode so I included it, but maybe it's not involved in that code path?

Reply:
It shouldn't be, the proxy information (host and required) is added in the 'extra' subkey and not part of the resource attributes.